### PR TITLE
Fix intermittent failures in test_connection

### PR DESCRIPTION
**Pull request by sridhargaddam**
Fixes issue: https://github.com/submariner-io/shipyard/issues/309

Signed-Off-by: Sridhar Gaddam <sgaddam@redhat.com>

### CODE REVIEW

**Review by mangelajo — APPROVED** [review]

**Review by mangelajo — COMMENTED** [review]
I would add also a check with 

subctl show connections 

to make sure we at least have one connection (it'd be better to check that we have the same amount as "joined" clusters if we can get that number somehow...) but not sure if it's easy to get.

**Comment by sridhargaddam** [comment]
> I would add also a check with
> 
> subctl show connections
> 
> to make sure we at least have one connection (it'd be better to check that we have the same amount as "joined" clusters if we can get that number somehow...) but not sure if it's easy to get.

Okay @mangelajo. I'll update the PR. Thanks for reviewing.

File: scripts/shared/lib/deploy_funcs [hunk 1]
```diff
@@ -75,6 +83,19 @@ function connectivity_tests() {
     with_context "$target_cluster" remove_resource "${RESOURCES_DIR}/nginx-demo.yaml"
 }
 
+function verify_gw_status() {
+    sleep_duration=5
```
Comment (mangelajo) [inline scripts/shared/lib/deploy_funcs:87]:
```suggestion
    sleep_duration=5
    # helm doesn't use the operator yet, and connection status is based on the operator object
    if subctl show connections 2>&1 | grep "the server could not find the requested resource"; then
        return 0
    fi

```

**Review by skitt — APPROVED** [review]

**Comment by dfarrell07** [comment]
> Expected "Miguel Angel Ajo Pelayo <miguelangel@ajo.es>", but got "Miguel Angel Ajo Pelayo <majopela@redhat.com>".

Is your sign-off email differently configured than your commit author email @mangelajo?

**Review by dfarrell07 — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Comment by tpantelis** [comment]
DCO check failed. Also can you update the PR summary to add details as to the problem/solution?

**Comment by mangelajo** [comment]
I have handled the DCO & Summary as part of the squash

**Pull request merged**